### PR TITLE
[MediaBundle] Folder types as constants

### PR DESCRIPTION
**Pull request by SpadXIII**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | 

I noticed a difference between the FolderType rel-field and the ones added in the fixtures. Extracted all used values into a class with constants.

### CODE REVIEW

**Comment by sandergo90** [comment]
Hi @SpadXIII ! Thanks for your contribution. I think we don't need the extra FolderTypes class here. Can you just add it to the Folder entity?

Thanks!

**Comment by SpadXIII** [comment]
I have moved the constants to the Folder entity

File: src/Kunstmaan/MediaBundle/DataFixtures/ORM/FolderFixtures.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ public function load(ObjectManager $manager)
 
         $subgal = new Folder();
         $subgal->setParent($gal);
-        $subgal->setRel('image');
+        $subgal->setRel(Folderypes::IMAGE);
```
Comment (acrobat) [inline src/Kunstmaan/MediaBundle/DataFixtures/ORM/FolderFixtures.php]: Incorrect class

File: src/Kunstmaan/MediaBundle/Entity/Folder.php [hunk 2]
```diff
@@ -23,6 +23,12 @@
  */
 class Folder extends AbstractEntity implements GedmoNode
 {
+    const FILES = 'files';
```
Comment (acrobat) [inline src/Kunstmaan/MediaBundle/Entity/Folder.php]: I would name them `TYPE_...`

File: src/Kunstmaan/MediaBundle/DataFixtures/ORM/FolderFixtures.php [hunk 3]
```diff
@@ -22,7 +22,7 @@ class FolderFixtures extends AbstractFixture implements OrderedFixtureInterface
     public function load(ObjectManager $manager)
     {
         $gal = new Folder();
-        $gal->setRel('media');
+        $gal->setRel(Folderypes::MEDIA);
```
Comment (acrobat) [inline src/Kunstmaan/MediaBundle/DataFixtures/ORM/FolderFixtures.php]: incorrect class

**Pull request merged**